python/lsst/ts/Dome/mock_controller.py

```diff
@@ -311,24 +327,44 @@ async def move_el(self, position):
         ----------
         position: `float`
             Desired elevation, in radians, in range [0, pi/2)
+
+        Returns
+        -------
+        `float`
+            The estimated duration of the execution of the command.
         """
         self.log.info(f"Received command 'moveEl' with argument position={position}")
 
         # No conversion from radians to degrees needed since both the commands
         # and the mock az controller use radians.
-        await self.lwscs.moveEl(position)
+        return await self.lwscs_status.moveEl(position, self.current_tai)
```

Comment:
You asked for clarification of my suggestion for not duplicating APIs. I am suggesting you delete this method and others like it, and call the mock subsystem methods directly. For example if you delete this method, then you replace calls to:
```
await self.moveEl(position)
```
with:
```
await self.lwscs_status.moveEl(position, self.current_tai)
```

Reply:
Clear, thanks. I'll look into that.

Reply:
That's fair.